### DISCUSSION TITLE
Rendre un acteur invisible en mode portrait

### DISCUSSION THREAD

**Discussion by Synfactory** [witsa/synapps#735] [Q&A] [answered]
Salut,
je souhaiterai pouvoir rendre un acteur invisible losque je passe en mode portrait. Je ne sais pas trop comment procéder.
Est ce qu'il est possible de récupérer la valeur du vmin et de rendre cet acteur invisible si le vmin est inférieur à une valeur?

Merci

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,
C'est encore bien plus simple.
Dans les liaisons il y a l'information du mode d'affichage
![image](https://github.com/witsa/synapps/assets/33632664/b25daaec-9ce6-4139-8a09-9baa7fb6ddf8)

**Reply by Synfactory** [reply]
:o Merci Nicolas
